Automattic/jetpack
# SSO: Implement users table user invite revoking [Original]

Fixes #

## Proposed changes:

* These changes build on the users table improvements from: https://github.com/Automattic/jetpack/pull/35158.
* The logic to revoke user invites has been added to the users table row actions for each user that has a 'pending invite'.

<img width="758" alt="CleanShot 2024-01-25 at 22 49 56@2x" src="https://github.com/Automattic/jetpack/assets/10482592/e55f1f96-bee1-4a6e-b35a-13fc0e87bbce">


### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






1. On an Atomic, JN, or whatever non-simple site, install Jetpack.
2. Make sure SSO is on, it should be on by default.
3. Sync this PR to your site.
4. Go to the users table (`wp-admin/users/php`).
5. It should have the status of the user's connection "Connected", "Invite", or "Pending invite".
6. Clicking "Invite" should invite the user and change their status to "Pending invite".
7. If you haven't already, add and invite a few new users to your site.
8. Hover the row actions and verify that rows with 'pending invite' only display the `Revoke invite` action.
9. Verify the 'Revoke invite' row action successfully revokes a user's site invitation.
